overlay/files/system/oem/11_persistency.yaml

```diff
@@ -13,6 +13,7 @@ stages:
           /etc/modprobe.d
           /etc/rancher
           /etc/sysconfig
+          /etc/profile.d
```

Comment:
I think we don't even need that but I've added while testing - I'll double check